tests/Hedgehog.Tests/GenTests.fs

```diff
@@ -30,18 +42,26 @@ let genTests = testList "Gen tests" [
         let ticks =
             Random.integral range
             |> Random.run seed1 0
-        let expected = System.DateTime ticks
 
-        let actual = Gen.dateTime (Range.constant System.DateTime.MinValue System.DateTime.MaxValue)
+        let actual =
+            (Range.constant System.DateTime.MinValue System.DateTime.MaxValue)
+            |> Gen.dateTime
+            |> Gen.toRandom
+            |> Random.run seed0 0
+            |> Tree.outcome
 
-        let result = actual |> Gen.toRandom |> Random.run seed0 0 |> Tree.outcome
-        expected =! result
+        let expected =
+            System.DateTime ticks
+        expected =! actual
 
     testCase "dateTime shrinks to correct mid-value" <| fun _ ->
-        let result =
+        let actual =
             property {
                 let! actual =
-                  Range.constantFrom (System.DateTime (2000, 1, 1)) System.DateTime.MinValue System.DateTime.MaxValue
+                  (Range.constantFrom
+                       (System.DateTime (2000, 1, 1))
+                        System.DateTime.MinValue
+                        System.DateTime.MaxValue)
```

Comment:
I think we can open `System` and not fully qualify `System.DateTime`. If `DateTime` is going to collide, we can use a type alias.

Reply:
Done. I didn't see any issues.